frontend/src/metabase/public/components/widgets/DisplayOptionsPane.styled.jsx

```diff
@@ -18,3 +19,13 @@ export const DisplayOption = styled.div`
 export const DisplayOptionTitle = styled.h3`
   margin-bottom: 1rem;
 `;
+
+export const ToggleContainer = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+export const ToggleLabel = styled.label`
+  margin-right: ${space(3)};
```

Comment:
Not exactly the space set in the Figma file which is `49px`. But I think using our design system util would be better.